src/containers/PsychoAssess/index.css

```diff
@@ -0,0 +1,51 @@
+.questionAnswers {
+  padding-left: 40px !important;
+  font-size: 20px !important;
+  color: #161b50;
+}
+
+.questionAnswers input {
+  margin-top: 13px;
+}
+
+.questionTitle {
+  color: rgb(0, 72, 145);
+  font-weight: bold;
+}
+
+label,
+input {
```

Comment:
we shouldn't use ```label, input``` like this bc it will affect others work, can you put their class name or whatever container before them that the changes will only be for these specific items